docs/conditions.md

```diff
@@ -1,144 +1,100 @@
-# Conditions
```

Comment:
Hi @jsabrooke i'm not sure what's happened to this page...the whole `Conditions` section seems to have been removed.

Reply:
Oh god oops! I copied the whole thing out for editing, then must have copied it back in and overwritten everything in the original file instead of just the cascade mode section...